lib/expeditor/command.rb

```diff
@@ -37,18 +45,18 @@ def start_with_retry(retryable_options = {})
     end
 
     def started?
-      @normal_future.executed?
+      !!@normal_future && @normal_future.executed?
```

Comment:
I was confused. The implementation doesn't return a future. I'll change it to avoiding `!!`.